# Description of the speed limitation on the map

uint64  id
float64 s_start
float64 s_end
float64 v_upper
float64 v_lower